code/frontend/package.json

```diff
@@ -30,19 +30,15 @@
         "uuid": "^9.0.1"
     },
     "devDependencies": {
-        "@types/chai": "^4.3.14",
         "@types/lodash-es": "^4.17.12",
-        "@types/mocha": "^10.0.6",
         "@types/node": "^20.12.5",
         "@types/react": "^18.2.74",
         "@types/react-dom": "^18.2.24",
         "@types/uuid": "^9.0.8",
         "@vitejs/plugin-react": "^4.2.1",
-        "chai": "^5.1.0",
-        "mocha": "^10.4.0",
         "prettier": "^3.2.5",
-        "ts-node": "^10.9.2",
         "typescript": "^5.4.4",
-        "vite": "^5.2.8"
+        "vite": "^5.2.8",
+        "vitest": "^1.5.0"
```

Comment:
I was able to get rid of a bunch of dependencies now that we're using vitest and not mocha.